README.md

```diff
@@ -55,16 +55,5 @@ Here are some common algorithms that can be applied to almost all data problems:
 
 ### SORTING
   - [Bubble Sort](https://github.com/TheAlgorithms/R/blob/master/sorting/Bubble%20sort.R)
+  - [Comb Sort](https://github.com/TheAlgorithms/R/blob/master/sorting/CombSort.R)
   - [Insertion Sort](https://github.com/TheAlgorithms/R/blob/master/sorting/Insertion%20sort.R)
-  - [Quick Sort](https://github.com/TheAlgorithms/R/blob/master/sorting/Quick%20sort.R)  
```

Comment:
This commit will delete content (13 lines from this point), you need to just add the link to comb sort to the existing [readme content](https://github.com/TheAlgorithms/R/blob/master/README.md).

Reply:
I'm sorry I do not fully understand what you are saying. I copied the existing README content and added a link.

Reply:
Your previous commits resulted into deletions (looking at file change). I guess you got it now since your latest commits corrected that.